brickstrap.sh

```diff
@@ -404,7 +440,10 @@ case "${cmd}" in
     copy-root)           copy-root;;
     configure-packages)  configure-packages;;
     run-hook)            run-hook ${BOARDDIR}/hooks/${run_hook_arg};;
-    run-hooks)           run-hooks;;
+    run-hooks)
+        run-hooks
+        dump-info-hook
```

Comment:
Since this is run at the same time as other hooks, why not just use a hook for this? I'm actually already doing something like this to generate a changelog template.

We could make sure hook files run in a deterministic order based on filename so that the `zzz-dump-info` would be guaranteed to run last. Of course we could add numeric prefixes to all of the files for better style, then we could have `99-dump-info`.